examples/kubernetes/templates/v1/cilium-crio-ds.yaml.sed

```diff
@@ -295,6 +213,11 @@ spec:
           defaultMode: 420
           optional: true
           secretName: cilium-clustermesh
+        # To read the configuration from the config map
+      - configMap:
+          defaultMode: 420
```

Comment:
Any particular reason for `420` (which I follow as user can read, group can write)?

Reply:
no, I copy paste it from the above, I'll remove it